mllm/backends/qnn/QNNUtils.cpp

```
@@ -298,6 +298,129 @@ bool freeQnnTensors(Qnn_Tensor_t*& tensors, uint32_t numTensors) {
+    case QNN_DATATYPE_FLOAT_16:
+    case QNN_DATATYPE_SFIXED_POINT_16:
+    case QNN_DATATYPE_UFIXED_POINT_16: return 2;
+
+    case QNN_DATATYPE_INT_32:
+    case QNN_DATATYPE_UINT_32:
+    case QNN_DATATYPE_FLOAT_32:
+    case QNN_DATATYPE_SFIXED_POINT_32:
+    case QNN_DATATYPE_UFIXED_POINT_32: return 4;
+
+    case QNN_DATATYPE_INT_64:
+    case QNN_DATATYPE_UINT_64: return 8;
```

Clang (14.0.6)

[error] 8 is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers,-warnings-as-errors)